packages/mdctl-axon-tools/lib/StudyManifestTools.js

```diff
@@ -147,7 +147,7 @@ class StudyManifestTools {
     this.writeIssues(removedEntities)
     this.writePackage('study', extraConfig)
 
-    return { manifest, removedEntities }
+    return { manifest, removedEntities, mappingScript }
```

Comment:
hi Sharad, this is unrelated to MIG-105, this is already addressed in [Davide's PR](https://github.com/Medable/mdctl/pull/372/files)

This ticket is about removing the workaround in MenuConfigMapping class